pmm-app/src/pmm-update-panel/components/LastCheck/LastCheck.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { MouseEvent } from 'react';
+import { Button } from '@grafana/ui';
+
+import * as styles from './LastCheck.styles';
+
+interface LastCheckProps {
+  onCheckForUpdates: (e: MouseEvent) => void;
+  isLoading: boolean;
+  lastCheckDate: string;
+}
+
+export const LastCheck = ({ isLoading, lastCheckDate, onCheckForUpdates }: LastCheckProps) => {
+  const icon = `fa ${isLoading ? 'fa-spin ' : ''}fa-refresh`;
```

Comment:
Grafana has its own spinner, we could use them here. The plan is to get rid fontawesome :)

Reply:
Grafana's documentation says that the `Icon` component is a wrapper around [Font Awesome and Unicons icons](https://developers.grafana.com/ui/latest/index.html?path=/docs/docs-overview-icon--icons-overview) (and I guess that applies also for the `Button` component and so on)

Reply:
That's not the point. The point is sustainability, which we won't achieve if we use classes like 'fa-spin'.